annotations/quadriga.py

```diff
@@ -275,3 +253,112 @@ def parse_response(raw_response):
         tag = child.tag.replace(QDNS, '')
         data[tag] = child.text
     return data
+
+
+
+def build_concept_node(concept, user, creation_time, source_uri):
+    """
+    Helper function to build a concept node dictionary.
+    """
+    return {
+        "label": concept.label or "",
+        "metadata": {
+            "type": "appellation_event",
+            "interpretation": concept.uri,
+            "termParts": [
```

Comment:
there should be one term part for every piece of annotated text in an appellation (I think currently there can only be one term part as vogon doesn't allow highlighting multiple parts that are not together for one appellation). Position needs to be the position of the first character of the term part.